components/organisms/ContributorProfileTab/contributor-profile-tab.tsx

```diff
@@ -164,6 +201,90 @@ const ContributorProfileTab = ({
+            role="button"
+            className="absolute right-5 top-5 text-xl text-slate-600"
+          />
+          <div className="flex-1 md:flex-[2.5]">
+            <div className="flex items-center gap-2">
+              <Image className="rounded" alt="Open Sauced Logo" width={30} height={30} src={openSaucedImg} />
+              <Title className="font-semibold text-lg" level={4}>
+                Do you know {login}?
+              </Title>
+            </div>
+
+            <p className="text-slate-500 text-sm mt-2">
+              Invite {login} to join OpenSauced to be able to access insights, interact with other developers and find
+              new open source opportunities!
+            </p>
+          </div>
+          <div className="flex items-end flex-col gap-2 self-end flex-1 max-md:w-full">
+            {!showSocialLinks && (
+              <Button
+                onClick={handleInviteClick}
+                className="max-md:w-full md:w-40 flex justify-center"
+                variant="primary"
+              >
+                Invite to opensauced
+              </Button>
+            )}
+
+            {showSocialLinks && (
+              <div className="flex items-center gap-3">
+                {twitter_username && (
+                  <a
+                    href={`https://twitter.com/intent/tweet?text=${encodeURIComponent(
+                      `Check out @saucedopen. The platform for open source contributors to find their next contribution. https://opensauced.pizza/blog/social-coding-is-back. @${twitter_username}`
+                    )}&hashtags=opensource,github`}
+                    target="_blank"
+                    rel="noopener noreferrer"
+                    className="text-white bg-blue-400 rounded-full p-3"
+                  >
+                    <FaXTwitter className="text-lg" />
+                  </a>
+                )}
+                {!!display_email && (
+                  <a
+                    href={`mailto:${email}?subject=${encodeURIComponent(
+                      "Invitation to join OpenSauced!"
+                    )}&body=${encodeURIComponent(emailBody)}`}
+                    className="text-white bg-red-400 rounded-full p-3"
+                  >
+                    <TbMailFilled className="text-lg" />
+                  </a>
+                )}
+                {!!linkedin_url && (
+                  <a
+                    href={`https://www.linkedin.com/in/${linkedin_url}`}
+                    className="text-white bg-blue-600 rounded-full p-3"
+                  >
+                    <FaLinkedinIn className="text-lg" />
+                  </a>
+                )}
+              </div>
+            )}
+
+            {!user && !showSocialLinks && (
+              <Button
+                onClick={() =>
+                  signIn({
+                    provider: "github",
+                    options: { redirectTo: `${window.location.origin}/user/${login}` },
```

Comment:
refactor: no need to add the full URL, or is this because of OAuth?

```suggestion
                    options: { redirectTo: `/user/${login}` },
```

Reply:
We need the origin because of OAuth. Supabase uses it to redirect the user back to their location before last location before authentication.